---
layout: default
---

# Centromeres and why dna sequence alone can't answer life's questions
Okay, centromeres. The **center** of the chromosome. \
One thing I've learnt over my experience in a molecular biology lab, reading about biology, is that there is always exceptions. Inorganic chemistry style, but a little more nuanced. Let me explain. You somehow muster up the skills to extract dna good enough to make fragmented sequences, and send them to a sequencing core (or sequence them yourself, either works). You get back your dna sequences. As a fasta file. In the process, you've lost several layers of information. you've lost the chromosome's 3-D structure present in the nucleus/protoplasm of whatever you obtained the dna sample from. You lost the methylation iformation attached to these sequences. And yet, there's so many projects whose sole aim is to sequence entire genomes and put them in a database. 
#### Why?
To put it simply, that's the best we can do. The amount of storage neededed is enormous, if we wish to capture 3-D interactions and store higher dimension sequencing data. And that's just the beginning of the problem. The kind of mathematical modeling required to answer these questions have not been scaled yet to the levels that's necessary, and it would definitely be overwhelming to grind this much data to obtain meaningful information. 
#### Why does the title of this post have centromeres?
This site is named after telomere ends, and I thought it best to explain a bit about centromeres, the other end (the middle, depends on how you look at it) of the chromosome sequence spectrum. I call it the other end for many reasons. For one, telomeres are a lot more defined on a sequence level than centromeres. You have regular patterns of repeated structures in telomeres, and life is good. And then you encounter centromeres. For a bioinformatician, centromeres are not fun to work with. 
